mindocr/utils/callbacks.py

```diff
@@ -351,3 +348,9 @@ def on_train_end(self, run_context):
         if self.is_main_device:
             self.rec.save_curves()  # save performance curve figure
             self.logger(f'=> Best {self.main_indicator}: {self.best_perf} \nTraining completed!')
+
+            if self.ckpt_save_policy == 'top_k':
+                log_str = f'Top K checkpoints:\n{self.main_indicator}\tcheckpoint\n'
+                for p, ckpt_name in self.ckpt_manager.get_ckpt_queue():
+                    log_str += f'{p:.4f}\t{os.path.join(self.ckpt_save_dir, ckpt_name)}\n'
+                self.logger(log_str)
```

Comment:
Showing the top k results during training is helpful to tell whether it is already overfitting.